Core/Object Arts/Dolphin/IDE/PublishedAspectInspectorTest.cls

```diff
@@ -240,6 +240,11 @@ testAspectChanged
 	self assert: treePresenter view itemCount == 1.
 	self repaint!
 
+testCommandMenuItemEditor
+	"#show reproduces the error and is less complicated to debug/test than #showModal"
```

Comment:
This comment won't be clear to someone who doesn't have the context of the issue, especially as time goes on.